### DISCUSSION TITLE
Can this repo for the periodic motion and orientation?

### DISCUSSION THREAD

**Discussion by HongminWu** [dfki-ric/movement_primitives#64] [Q&A]
Thanks for sharing.  Though DMPs are widely used to encode point-to-point movements, implementing the periodic DMP for translation and orientation is still challenging. Can this repository achieve these? If possible, would you provide any examples?

**Comment by AlexanderFabisch** [comment]
Hi @HongminWu ,

periodic DMPs are currently not implemented in this library. I don't have the resources to implement it, but would be open for contributions in case you are interested.
